net2/BroDetect.js

```diff
@@ -654,35 +654,35 @@ module.exports = class {
             //log.error("Conn:Diff:",obj.proto, obj.resp_ip_bytes,obj.resp_pkts, obj.orig_ip_bytes,obj.orig_pkts,obj.resp_ip_bytes-obj.resp_bytes, obj.orig_ip_bytes-obj.orig_bytes);
             if (obj.resp_bytes >100000000) {
                 if (obj.duration<1) {
-                    log.error("Conn:Burst:Drop",obj);
+                    log.debug("Conn:Burst:Drop",obj);
                     return;
                 }
                 let rate = obj.resp_bytes/obj.duration;
                 if (rate>20000000) {
-                    log.error("Conn:Burst:Drop",rate,obj);
+                    log.debug("Conn:Burst:Drop",rate,obj);
                     return;
                 }
                 let packet = obj.resp_bytes/obj.resp_pkts;
                 if (packet >10000000) {
-                    log.error("Conn:Burst:Drop2",packet,obj);
+                    log.debug("Conn:Burst:Drop2",packet,obj);
                     return;
                 }
             }
 
 
             if (obj.orig_bytes >100000000) {
                 if (obj.duration<1) {
-                    log.error("Conn:Burst:Drop:Orig",obj);
+                    log.debug("Conn:Burst:Drop:Orig",obj);
                     return;
                 }
                 let rate = obj.orig_bytes/obj.duration;
                 if (rate>20000000) {
-                    log.error("Conn:Burst:Drop:Orig",rate,obj);
+                    log.debug("Conn:Burst:Drop:Orig",rate,obj);
                     return;
                 }
                 let packet = obj.orig_bytes/obj.orig_pkts;
                 if (packet >10000000) {
-                    log.error("Conn:Burst:Drop2:Orig",packet,obj);
+                    log.debug("Conn:Burst:Drop2:Orig",packet,obj);
                     return;
```

Comment:
there errors need to be there.  they rarely show and when show it helps us find problems